autoload/mysql.vim

```diff
@@ -107,7 +107,16 @@ endfunction
 "Use this function to customise
 "the removal of said crap...
 function! mysql#PostBufferFormat()
-    keepjumps normal! gg"_2dd
+    " keepjumps normal! gg"_2dd
+    if(&ft == 'testType')
+        return
+    endif
+    if(&ft == 'SQHInsert')
```

Comment:
I think these are modifying the jump list, meaning if I do `<c-o>` or `<c-i>` I need to press them a few times, can we avoid adding to the jump list?